apps/gallery/index.html

```diff
@@ -57,7 +57,6 @@
     <link rel="stylesheet" type="text/css" href="app://theme.gaiamobile.org/shared/elements/gaia-theme/gaia-theme.css">
     <link rel="stylesheet" type="text/css" href="shared/elements/gaia-icons/gaia-icons.css" />
     <link rel="stylesheet" type="text/css" href="shared/elements/gaia-icons/bidi-helper.css" />
-    <script defer src="shared/elements/gaia-header/dist/gaia-header.js"></script>
```

Comment:
Don't you need to put this line in a comment to ensure that it gets packaged correctly?

I'm not sure why this patch actually works, unless we're packaging way too much shared/ stuff into the application.zip file...